tests/disaster-recovery/sc_arbiter/test_shutdowns.py

```diff
@@ -0,0 +1,487 @@
+        log.info("Running logwriter cephFS and RBD workloads")
+        (
+            sc_obj.cephfs_logwriter_dep,
+            sc_obj.cephfs_logreader_job,
+        ) = setup_logwriter_cephfs_workload_factory(read_duration=0)
+
+        # Generate 5 minutes worth of logs before inducing the netsplit
+        log.info("Generating 5 mins worth of log")
+        time.sleep(300)
+
+        sc_obj.get_logwriter_reader_pods(label=constants.LOGWRITER_CEPHFS_LABEL)
+        sc_obj.get_logwriter_reader_pods(label=constants.LOGREADER_CEPHFS_LABEL)
+        sc_obj.get_logwriter_reader_pods(
+            label=constants.LOGWRITER_RBD_LABEL, exp_num_replicas=2
+        )
+
+        zones = constants.ZONES_LABELS
+        zones.remove("arbiter")
+
+        for i in range(iteration):
+            log.info(f"------ Iteration {i+1} ------")
+
+            # note the file names created
+            sc_obj.get_logfile_map(label=constants.LOGWRITER_CEPHFS_LABEL)
+            sc_obj.get_logfile_map(label=constants.LOGWRITER_RBD_LABEL)
+
+            # Fetch the nodes in zone that needs to be crashed
+            zone = random.choice(zones)
+            nodes_to_shutdown = sc_obj.get_nodes_in_zone(zone)
+
+            assert (
+                len(nodes_to_shutdown) != 0
+            ), f"There are 0 zone nodes labeled as topology.kubernetes.io/zone={zone}!!"
+
+            # crash zone nodes
+            log.info(f"Crashing zone {zone}")
+            thread_exec = futures.ThreadPoolExecutor(max_workers=len(nodes_to_shutdown))
+            start_time = datetime.now(timezone.utc)
+            futures_obj = []
+            crash_cmd = "echo c > /proc/sysrq-trigger"
+            for node in nodes_to_shutdown:
+                futures_obj.append(
+                    thread_exec.submit(
+                        OCP().exec_oc_debug_cmd, node=node.name, cmd_list=[crash_cmd]
+                    )
+                )
+                log.info(f"Crashed {node.name}")
+
+            # wait for the crash tasks to complete
+            log.info("Wait for the crash tasks to complete!")
+            futures.wait(futures_obj)
+
+            # delete debug pods if not deleted already
+            debug_pods = get_debug_pods([node.name for node in nodes_to_shutdown])
+            for pod in debug_pods:
+                try:
+                    pod.delete()
+                except CommandFailed:
+                    continue
+                else:
+                    log.info(f"Deleted pod {pod.name}")
+
+            # wait for the nodes to come back to READY status
+            log.info("Waiting for the nodes to come up automatically after the crash")
+            retry(
+                (
+                    CommandFailed,
+                    TimeoutError,
+                    AssertionError,
+                    ResourceWrongStatusException,
+                ),
+                tries=30,
+                delay=15,
+            )(wait_for_nodes_status(timeout=1800))
+
```

Comment:
we dont need to wait for the rebalance ?

Reply:
no we dont have to wait for re-balance, since we are inducing realistic crash scenario where there will be no wait. We wait for node status to get that correct note of end time when the crash has stopped.